unstructured/ingest/logger.py

```diff
@@ -1,12 +1,81 @@
+import ast
+import json
 import logging
 
 logger = logging.getLogger("unstructured.ingest")
 
 
+def hide_sensitive_fields(data: dict) -> dict:
```

Comment:
Can you make a simple unit test for this function and extract_json. Would help others to see exactly whats going on a bit quicker.